src/pubsub.c

```diff
@@ -615,6 +615,73 @@ void publishCommand(client *c) {
     addReplyLongLong(c,receivers);
 }
 
+void pubsubReplySubscribersClientInfo(client *c, list *clients) {
+    listNode *ln;
+    listIter li;
+    listRewind(clients, &li);
+    while ((ln = listNext(&li)) != NULL) {
+        client *client = listNodeValue(ln);
+        if (client->name) {
+            addReplyMapLen(c, 2);
+            addReplyBulkCString(c, "name");
+            addReplyBulk(c, client->name);
+        } else {
+            addReplyMapLen(c, 1);
+        }
+        addReplyBulkCString(c, "id");
+        addReplyBulkLongLong(c, client->id);
+    }
+}
+
+void pubsubReplySubscribers(client *c) {
+    sds type = c->argv[2]->ptr;
+    dict *d = NULL;
+    if (!strcmp(type, "global")) {
```

Comment:
```suggestion
    if (!strcasecmp(type, "global")) {
```
I assume we aren't case sensitive here.